next.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
any reason for this @digant2482 ?
I think strictMode should be true!

Reply:
Again during tests, strictMode sometimes is hard to work. I was sending authentication codes request twice, and since the codes were one-time usable, the second request was always failing. Thank you for pointing though, will remove this as well